### PR TITLE
USHIFT-3476: Test harness ipv6 adaptations

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

**Comment by pacevedom** [comment]
/hold
just found an error coming from https://github.com/openshift/microshift/pull/3636

File: test/bin/scenario.sh [hunk 1]
```diff
@@ -83,12 +83,31 @@ copy_file_to_vm() {
...
+    local -r vmname="$1"
+    local -r remote_filename="$2"
+    local -r local_filename="$3"
+
+    local ip
+    ip=$(get_vm_property "${vmname}" ip)
+    if [ "${ip}" != "${ip#*:[0-9a-fA-F]}" ]; then
+      ip="[${ip}]"
+    fi
+    local -r ssh_port=$(get_vm_property "${vmname}" ssh_port)
```
Comment (copejon) [inline test/bin/scenario.sh:106]: It's probably not likely to fail, but it'd be good to follow the same pattern used for `ip` to make sure the exit status of the subproc is caught.

**Comment by copejon** [comment]
It looks good. A minor nit, but not a blocker. I'll label and hold incase you want to fix it, else feel free to drop the hold

/lgtm
/hold

File: test/bin/scenario.sh (hunk 1 above)
Comment (pacevedom) [reply test/bin/scenario.sh:106]: Created https://issues.redhat.com/browse/USHIFT-3769 for this purpose. Thanks!

**Pull request merged**